### ISSUE TITLE
✅ 🔗 🐞 Bug Report: Editor Top Bar: Display Conditions are missing from the Publish dropdown for templates (Related to #21675)

### ISSUE THREAD

**Issue by AGHiggins** [elementor/elementor#22050]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When editing an existing template e.g. header, the Display Conditions are not available in the Publish dropdown when the new Editor Top Bar is being used. After disabling Editor Top Bar, Display Conditions are listed again.

### Steps to reproduce

1. Enable Editor Top Bar.
2. Create a template e.g. header.
3. Publish and set your display conditions.
4. Reopen the template to edit it.
5. Click the dropdown arrow next Publish. Note 'Display Conditions' are not shown.
6. Disable Editor Top Bar.
7. Open the template again and now 'Display Conditions' are available.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: mariadb.org binary distribution v10.6.12
	PHP Version: 8.0.28
	PHP Memory Limit: -1
	PHP Max Input Vars: 5000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	Site URL: https://jagodagems-ye1wndt9gf.live-website.com
	Home URL: https://jagodagems-ye1wndt9gf.live-website.com
	WP Multisite: No
	Max Upload Size: 64 MB
	Memory limit: 40M
	Max Memory limit: -1
	Permalink Structure: /%postname%/
	Language: en-GB
	Timezone: 2
	Admin Email: andy@chisenga.com
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.12.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.12.2
		Author: Elementor.com


== Must-Use Plugins ==
	IONOS Marketplace
		Version: 1.1.5
		Author: IONOS

	WordPress automation by Installatron
		Version: 
		Author: 


== Elements Usage ==
	
	footer : 1
		form : 1
		heading : 3
		image : 1
	header : 2
		container : 1
		divider : 1
		image : 1
		jet-nav-menu : 1
	popup : 1
		container : 2
		form : 1
		heading : 1
		image : 1
	wp-page : 1
		button : 3
		container : 13
		divider : 1
		heading : 5
		image : 5
		jet-nav-menu : 1
		posts : 1
		shortcode : 1
		text-editor : 4


== Settings ==
	
	cpt_support: post, page, e-landing-page
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: swap


== Features ==
	Custom Fonts: 1
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimised DOM Output: Active by default
	Improved asset loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Editor Top Bar: Active
	Landing pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default


== Log ==
	
PHP: showing 9 of 9PHP: 2022-11-12 18:17:03 [notice X 11][/Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor-pro/core/utils/collection.php::75] Return type of ElementorPro\Core\Utils\Collection::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-11-12 18:23:51 [notice X 47][/Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor-pro/core/database/model-base.php(10): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor-pro/plugin.php(153): class type include()array (
  0 => '/Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor-pro/core/database/model-base.php',
)
#2: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php(18): ElementorPro\Plugin -> autoload()
#3: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor-pro/plugin.php(153): class type include()array (
  0 => '/Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php',
)
#4: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor-pro/modules/notes/data/controller.php(243): ElementorPro\Plugin -> autoload()
',
)]
PHP: 2022-11-12 18:23:51 [notice X 5][/Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::304] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(304): class type trigger_error()
#2: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/bdthemes-element-pack/includes/controls/select-input/dynamic-select.php(124): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2022-11-12 18:23:51 [notice X 5][/Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::304] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(304): class type trigger_error()
#2: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/bdthemes-element-pack/loader.php(716): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2022-11-12 18:23:51 [notice X 5][/Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::304] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(304): class type trigger_error()
#2: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/bdthemes-element-pack/loader.php(724): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2022-11-12 18:49:28 [notice X 25][/Users/andyhiggins/Local Sites/mylocal/app/public/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::304] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-20 14:14:00 [notice X 435][/Users/andyhiggins/Local Sites/jagoda/app/public/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::200] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-23 16:59:00 [error X 1][/Users/andyhiggins/Local Sites/jagoda/app/public/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::295] Uncaught Error: Call to a member function update_meta() on null in /Users/andyhiggins/Local Sites/jagoda/app/public/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:295
Stack trace:
#0 /Users/andyhiggins/Local Sites/jagoda/app/public/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(168): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(5, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /Users/andyhiggins/Local Sites/jagoda/app/public/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /Users/andyhiggins/Local Sites/jagoda/app/public/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 /Users/andyhiggins/Local Sites/jagoda/app/public/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#5 /Users/andyhiggins/Local Sites/jagoda/app/public/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /Users/andyhiggins/Local Sites/jagoda/app/public/wp-admin/admin-ajax.php(188): do_action('wp_ajax_element...')
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-04 09:19:32 [error X 1][/Users/andyhiggins/Local Sites/jagoda/app/public/wp-content/plugins/elementor-pro/modules/popup/module.php::122] Uncaught Error: Call to undefined method Elementor\Core\Kits\Documents\Kit::save_display_settings_data() in /Users/andyhiggins/Local Sites/jagoda/app/public/wp-content/plugins/elementor-pro/modules/popup/module.php:122
Stack trace:
#0 [internal function]: ElementorPro\Modules\Popup\Module->save_display_settings(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#1 /Users/andyhiggins/Local Sites/jagoda/app/public/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /Users/andyhiggins/Local Sites/jagoda/app/public/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#3 /Users/andyhiggins/Local Sites/jagoda/app/public/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#4 /Users/andyhiggins/Local Sites/jagoda/app/public/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /Users/andyhiggins/Local Sites/jagoda/app/public/wp-admin/admin-ajax.php(188): do_action('wp_ajax_element...')
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 20 of 302023-03-23 16:00:54 [info] elementor-pro::elementor_pro_updater Started 
2023-03-23 16:00:54 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-03-23 16:00:54 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-03-23 16:00:54 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.6',
  'to' => '3.11.7',
)]
2023-03-23 16:00:54 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.6',
  'to' => '3.11.7',
)]
2023-04-04 07:29:30 [info] elementor::elementor_updater Started 
2023-04-04 07:29:30 [info] Elementor/Upgrades - _on_each_version Start  
2023-04-04 07:29:30 [info] Elementor/Upgrades - _on_each_version Finished 
2023-04-04 07:29:30 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.5',
  'to' => '3.12.1',
)]
2023-04-04 07:29:30 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.5',
  'to' => '3.12.1',
)]
2023-04-04 07:30:21 [info] elementor-pro::elementor_pro_updater Started 
2023-04-04 07:30:22 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-04-04 07:30:22 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-04 07:30:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.7',
  'to' => '3.12.1',
)]
2023-04-04 07:30:22 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.7',
  'to' => '3.12.1',
)]
2023-04-14 14:15:33 [info] elementor-pro::elementor_pro_updater Started 
2023-04-14 14:15:33 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-04-14 14:15:33 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-14 14:15:33 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.1',
  'to' => '3.12.2',
)]
2023-04-14 14:15:33 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.1',
  'to' => '3.12.2',
)]

JS: showing 2 of 2JS: 2023-03-20 17:43:47 [error X 8][http://jagoda.local/wp-admin/load-scripts.php?c=1&loadchunk_0=jquery-core,jquery-migrate,underscore,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:2:51824] Cannot read properties of undefined (reading 'replace') 
JS: 2023-04-04 07:29:29 [error X 1][http://jagoda.local/wp-content/plugins/elementor/assets/js/import-export-admin.min.js?ver=3.11.5:2:5394] elementorImportExport is not defined 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Incompatible

== Elementor Pro - Compatibility Tag ==
	

				

````

</details>

**Comment by aperojo** [comment]
Same problem here, please fix it devs :)

**Comment by toineenzo** [comment]
Maybe its not a bug, but just not implemented yet?

**Comment by luarcito** [comment]
+1

**Comment by klentgary** [comment]
+1

**Comment by nicholaszein** [comment]
Hey everyone!

The new Top Bar is still an alpha preview, and not all features are available yet, including Display Conditions. But we have news!

We are happy to announce that this new feature is now live in the latest v3.13-Beta release!

We would love to see you try this new feature and give your feedback in the [Beta Announcement](https://github.com/orgs/elementor/discussions/22119)! 💪

Please try it out and let us know!

https://github.com/orgs/elementor/discussions/22119

Kind regards

PS: If you don't have time to test this out, please consider disabling the Top Bar feature until we release version 3.13.

**Comment by nicholaszein** [comment]
Hi everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.13.0**! 🥳 

You can now define your Display Conditions using the Top Bar dropdown button alongside the Update button.

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
